screen-editor.tex

```diff
@@ -96,6 +97,6 @@ \section{Editor Functionality}
 
 Long press on \widekey{RESTORE} to go into the Freeze Menu.  Then press \megakey{J} to switch joystick ports without having to physically swap the joystick to the other port.
 
-Go to \textbf{Fast mode} with \screentext{POKE 0,65} or use the Freeze Menu.
+Go to \textbf{Fast mode} with {\bf POKE 0,65} or use the Freezer. You can learn more about the Freezer on page \pageref{sec:freezer}.
```

Comment:
I kinda felt the prior use of \screentext here might have been trying to convey that the user should type it on the screen, though I don't know how prevalent that way of conveying this intention is. Not too fussed about the change to \bf, but maybe \screentext had the advantage of being monospaced font at least?